components/admin/requests/payment-information/Form.tsx

```diff
@@ -228,87 +166,69 @@ export default function PaymentDetailsForm({
+                name="paymentInformation.billingAddressLine1"
+                label="Address line 1"
+                required
+              >
+                <FormHelperText color="text.secondary">
+                  {'Street Address, P.O. Box, Company Name, c/o'}
+                </FormHelperText>
+              </TextField>
+            </Box>
+
+            <Box paddingBottom="24px">
+              <TextField name="paymentInformation.billingAddressLine2" label="Address line 2">
+                <FormHelperText color="text.secondary">
+                  {'Apartment, suite, unit, building, floor, etc'}
+                </FormHelperText>
+              </TextField>
+            </Box>
 
             <Stack direction="row" spacing="20px" paddingBottom="24px">
-              <FormControl isRequired>
-                <FormLabel>{'City'}</FormLabel>
-                <Input value={billingCity} onChange={handleChange('billingCity')} />
-              </FormControl>
+              <TextField name="paymentInformation.billingCity" label="City" required />
 
-              <FormControl>
-                <FormLabel>{'Province / Territory'}</FormLabel>
-                <Select
-                  placeholder="Select province / territory"
-                  value={billingProvince || undefined}
-                  onChange={event =>
-                    onChange({
-                      ...paymentInformation,
-                      billingProvince: event.target.value as Province,
-                    })
-                  }
-                >
-                  <option value={'ON'}>Ontario</option>
-                  <option value={'BC'}>British Columbia</option>
-                </Select>
-              </FormControl>
+              <SelectField
+                name="paymentInformation.billingProvince"
+                label="Province / Territory"
+                placeholder="Select province / territory"
+                required
+              >
+                <option value={'ON'}>Ontario</option>
+                <option value={'BC'}>British Columbia</option>
+              </SelectField>
             </Stack>
 
             <Stack direction="row" spacing="20px">
-              {/* TODO: Replace with dropdown */}
-              <FormControl isRequired>
-                <FormLabel>{'Country'}</FormLabel>
-                <Input value={billingCountry} onChange={handleChange('billingCountry')} />
-              </FormControl>
-              <FormControl isRequired>
-                <FormLabel>{'Postal code'}</FormLabel>
-                <Input
-                  value={billingPostalCode || ''}
-                  onChange={handleChange('billingPostalCode')}
-                />
+              <SelectField
+                name="paymentInformation.billingCountry"
+                label="Country"
+                required
+                placeholder="Select country"
+              >
+                {/* TODO: Add more countries? */}
```

Comment:
Hmm true, I think if we don't add more countries rn we maybe should just leave it as an `Input`. @OustanDing what are your thoughts?